draft-ietf-quic-http.md

```diff
@@ -598,11 +623,11 @@ HTTP/3 does not support the HTTP Upgrade mechanism (Section 6.7 of [RFC7230]) or
 
 ## Server Push
 
-Server push is an interaction mode introduced in HTTP/2 {{!HTTP2}} which permits
+Server push is an interaction mode introduced in HTTP/2 {{?HTTP2}} which permits
```

Comment:
Likewise, 

`Server push is an interaction mode that allows a server to push ...`

the history is not important (and also described at the end of para).